likelihood/testrun.py

```diff
@@ -29,3 +33,66 @@
 pf = {k: bf[k] for k in p0.keys()}
 print("Final params: ")
 print(pf)
+
+#======================DETERMINE ERRORS ON PARAMETERS========================
+
+# remove cobaya_out directory (just for now!) to make running code easier
+os.system('rm -r cobaya_out')  
+
+h = 0.005  # stepsize
+
+def fstep(params,signs):  # Determine likelihood at new steps 
+    newp0 = pf.copy()
```

Comment:
As a matter of principle, you shouldn't be using global variables in your function. Either pass pf as a parameter, or don't use it.